### DISCUSSION TITLE
ScriptReference/PopupWindow

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#239] [Notes]
# ScriptReference/PopupWindow

https://docs.unity3d.com/ScriptReference/PopupWindow.html

**Comment by mvi** [comment]
A slightly cleaner approach than needing to cache the last rect in a member field is to use `GUILayoutUtility.GetRect()` to get the next rect before you draw the button and then `GUI.Button(rect)` to draw your button which can then also trigger the PopupWindow by supplying the same rect.
